packages/cli/src/commands/doctor/healthchecks/common.ts

```diff
@@ -2,13 +2,28 @@ import chalk from 'chalk';
 import readline from 'readline';
 import wcwidth from 'wcwidth';
 import stripAnsi from 'strip-ansi';
+import {Ora} from 'ora';
 import {logger} from '@react-native-community/cli-tools';
 
 // Space is necessary to keep correct ordering on screen
-const logMessage = (message: string) => logger.log(`   ${message}`);
+const logMessage = (message?: string) => {
+  const indentation = '   ';
+
+  if (typeof message !== 'string') {
+    logger.log();
+
+    return;
+  }
+
+  const messageByLine = message.split('\n');
+
+  return logger.log(`${indentation}${messageByLine.join(`\n${indentation}`)}`);
+};
+
+const addBlankLine = () => logMessage();
```

Comment:
This is adding 2 blank lines (it shows on the screenshot a well), can we call empty `logger.log()`?

Reply:
It does call `logger.log()` on line `13` as no `message` is provided.

Reply:
With extra newline, resulting in 2 newlines instead of one.

Reply:
Oh, I missed that, sorry!